<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BagViewer</class>
 <widget class="QMainWindow" name="BagViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bagViewer</string>
  </property>
  <widget class="BagGL" name="bagGL">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuColormap">
     <property name="title">
      <string>&amp;Colormap</string>
     </property>
     <addaction name="actionTopographic"/>
     <addaction name="actionOmnimap"/>
    </widget>
    <widget class="QMenu" name="menuDraw_style">
     <property name="title">
      <string>&amp;Draw style</string>
     </property>
     <addaction name="actionSolid"/>
     <addaction name="actionWireframe"/>
     <addaction name="actionPoints"/>
    </widget>
    <addaction name="menuColormap"/>
    <addaction name="menuDraw_style"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionTopographic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Topographic</string>
   </property>
  </action>
  <action name="actionOmnimap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Omnimap</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Solid</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wireframe</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Points</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BagGL</class>
   <extends>QWidget</extends>
   <header>BagGL.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
